src/pages/index.js

```diff
@@ -3,20 +3,70 @@ import FormValidator from "../components/FormValidator.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import UserInfo from "../components/UserInfo.js";
 import PopupWithImage from "../components/PopupWithImage.js";
+import PopupWithFormConfirmDelete from "../components/PopupWithFormConfirmDelete.js";
 import Section from "../components/Section.js";
+import Api from "../components/Api.js";
 import "./index.css";
 
 import {
   config,
-  profileTitle,
-  profileDesc,
   inputTitle,
   inputDesc,
   editProfileButton,
-  initialCards,
   editProfileModalSelector,
+  changeProfilePicModalSelector,
+  profileImage,
 } from "../utils/constants.js";
 
+/* -------------------------------------------------------------------------- */
+/*                             API                                            */
+/* -------------------------------------------------------------------------- */
+
+const api = new Api({
+  baseUrl: "https://around.nomoreparties.co/v1/cohort-3-en",
+  headers: {
+    authorization: "dfa65752-2d8e-401e-8af5-014c787612d4",
+    "Content-Type": "application/json",
+  },
+});
+
+let myUserID = null;
+
+myUserID = api.whoAmI().then((response) => {
+  return response._id;
+});
+
+let cardList;
+
+api.getInitialCards("cards").then((cards) => {
+  api.whoAmI().then((response) => {
+    cardList = new Section(
+      {
+        items: cards,
+        renderer: (item) => {
+          const card = createCard(item, response._id);
+          cardList.addItem(card);
+        },
+      },
+      cardListSelector
+    );
+    cardList.renderItems();
+  });
+});
+
+const user = new UserInfo(
+  ".profile__title",
+  ".profile__description",
+  ".profile__image"
+);
+
+api.getUserInfo("users", "me").then((response) => {
```

Comment:
There is no need to have this api call. You should initialize the user when receiving all the initial data on line  42